test/e2e/config/containers.conf

```diff
@@ -52,3 +52,7 @@ dns_options=[ "debug", ]
 tz = "Pacific/Honolulu"
 
 umask = "0002"
+
+[engine]
+
+network_cmd_options=["allow_host_loopback=true"]
```

Comment:
Will this setting apply also to networks created by non-root users via `podman network create`?

Reply:
Yes, it should work